README.md

```diff
@@ -11,7 +11,7 @@
     <img src="https://img.shields.io/badge/licence-APL2-brightgreen"
          alt="license">
   </a>
-  <a href="https://github.com/kairos-io/provider-kairos/issues"><img src="https://img.shields.io/github/issues/kairos-io/provider-kairos"></a>
+  <a href="https://github.com/kairos-io/provider-kairos/v2/issues"><img src="https://img.shields.io/github/issues/kairos-io/provider-kairos"></a>
```

Comment:
This one does not need to have the v2 in it

Reply:
Ah, whoops, my bad!